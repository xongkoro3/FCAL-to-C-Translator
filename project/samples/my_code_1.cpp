#include <iostream>
#include "Matrix.h"
#include <math.h>

using namespace std;
int main () {
int x;
while(x < 7){
{
x = x + 1;
}}
int y;
y = x * 3;
int z;
z = x + 3;
cout << x ;
cout << "\n" ;
cout << y ;
cout << "\n" ;
cout << z ;
cout << "\n" ;
}

