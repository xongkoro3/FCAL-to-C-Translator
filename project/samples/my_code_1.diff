1,3c1,3
< 1751375926
< 959160482
< 1751375929
---
> 7
> 21
> 10
