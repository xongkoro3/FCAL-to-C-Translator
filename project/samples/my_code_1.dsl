/* This program is just a copy of sample_4.dsl. You should write your
   solution for my_code_1 in this file. */
/* This is our own test my_code_1 */

main () { 
  Int x;
  while(x < 7)
  {
    x = x + 1 ;
  }
  
  Int y;
  y = x * 3 ;


  Int z;
  z = x + 3 ;

  print(x) ;
  print("\n") ;
  print(y) ;
  print("\n") ;
  print(z) ;
  print("\n") ;  
}



