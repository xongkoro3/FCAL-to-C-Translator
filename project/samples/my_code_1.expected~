7
21
1000
