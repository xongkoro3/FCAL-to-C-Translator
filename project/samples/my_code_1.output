1751375926
959160482
1751375929
