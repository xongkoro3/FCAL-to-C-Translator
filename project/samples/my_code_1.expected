7
21
10
